src/NServiceBus.RabbitMQ/Routing/ISupportNativeDelays.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using global::RabbitMQ.Client;
+
+    /// <summary>
+    /// An extension for <see cref="IRoutingTopology"/> implementations which allows
+    /// for receiving messages from the delay infrastructure.
```

Comment:
Since the interface has the word "native" in it, should it also appear somewhere in this summary? I.e. the name of the interface suggests that there is also something called "non-native" delays.